packages/react-devtools-shared/src/devtools/views/Settings/DebuggingSettings.js

```diff
@@ -35,7 +37,18 @@ export default function DebuggingSettings(_: {||}) {
           Append component stacks to console warnings and errors.
         </label>
       </div>
-
+      <div className={styles.Setting}>
+        <label>
+          <input
+            type="checkbox"
+            checked={enableDoubleLogging}
+            onChange={({currentTarget}) =>
+              setEnableDoubleLogging(currentTarget.checked)
+            }
+          />{' '}
+          Enable double logging
```

Comment:
This is not a very clear name for an option. A React user wouldn't know what this means.

I think instead it should be something like "Suppress console during development-only second render pass" and true by default. I.e. invert the option.